.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: 2.1
+
+vars: 
+  - &cache_key raiden-wizard-{{ .Branch }}-{{ checksum "requirements.txt" }}
+
+executors:
+  python-executer:
+    working_directory: ~/src
+    docker:
+      - image: circleci/python:3.7
+
+jobs:
+  install:
+    executor: python-executer
+    steps:
+      - checkout
+      - restore_cache:
+          key: *cache_key
+      - run:
+          command: |
+            python3 -m venv venv
```

Comment:
In Raiden we check if the venv alreadz exists: https://github.com/raiden-network/raiden/blob/develop/.circleci/config.yml#L190

Reply:
I will add that. Thanks you!